packages/teams-js/test/public/pages.spec.ts

```diff
@@ -42,6 +42,21 @@ describe('Testing pages module', () => {
 
       Object.values(FrameContexts).forEach(context => {
         if (allowedContexts.some(allowedContexts => allowedContexts === context)) {
+          it(`pages.returnFocus should throw errors when pages is not supported when initialized with ${context}`, async () => {
+            await utils.initializeWithContext(context);
+            utils.setRuntimeConfig({ apiVersion: 1, supports: {} });
+            expect.assertions(1);
+            try {
+              pages.returnFocus();
+            } catch (e) {
+              expect(e).toEqual(errorNotSupportedOnPlatform);
+            }
+            utils.setRuntimeConfig({
+              apiVersion: 1,
+              supports: { pages: { appButton: {}, backStack: {}, config: {}, fullTrust: {}, tabs: {} } },
+            });
```

Comment:
Why is this code necessary if we're resetting the runtime config in the `afterEach` code on line 31?

Reply:
This comment applies to all the places we're doing this in the tests.

Reply:
I'll update them in the next commit.

Reply:
If you determine that it is required, can we add a helper method that resets the pages capabilities and subcapabilities?

Reply:
Thinking more about this...If I understand this correctly, this seems like something that should be done as part of test setup for each test that requires that the pages specific capability be supported in order to test the behavior.

Reply:
Sure, I'll work on it and get back to you.